packages/orbit-components/src/Accordion/Accordion.stories.js

```diff
@@ -0,0 +1,156 @@
+      <Accordion>
+        <AccordionSection header={<div>Default Accordion with header</div>}>
+          This is a section content
+        </AccordionSection>
+        <AccordionSection expandable={false} header={<div>Default Accordion with header</div>}>
+          This is a section content
+        </AccordionSection>
+        <AccordionSection expandable={false} header={<div>Default Accordion with header</div>}>
+          This is a section content
+        </AccordionSection>
+        <AccordionSection header={<div>Default Accordion with header</div>}>
+          This is a section content
+        </AccordionSection>
+        <AccordionSection expandable={false} header={<div>Default Accordion with header</div>}>
+          This is a section content
+        </AccordionSection>
+      </Accordion>
+    );
+  })
+  .add("Accordion with actions", () => {
+    const expanded = select("expanded", ["0X0", "0X1", "0X2"], "0X1");
+    return (
+      <Accordion expanded={expanded}>
+        <AccordionSection
+          id="0X0"
+          header={<div>This is a header label</div>}
+          actions={<Button>Open</Button>}
+        >
+          This is a content
+        </AccordionSection>
+        <AccordionSection
+          id="0X1"
+          header={<div>This is a header label</div>}
+          actions={<Button>Open</Button>}
+        >
+          This is a content
+        </AccordionSection>
+        <AccordionSection
+          id="0X2"
+          header={<div>This is a header label</div>}
+          actions={
+            <Stack direction="row">
+              <Button type="secondary">Edit</Button>
+              <Button>Custom close</Button>
+            </Stack>
+          }
+        >
+          Ja sam content 3
+        </AccordionSection>
+      </Accordion>
+    );
+  })
+  .add("Accordion with sticky footer", () => {
+    return (
+      <Accordion expanded="0X1">
+        <AccordionSection
+          id="0X0"
+          onExpand={action("onExpand")}
+          header={
+            <Stack justify="between">
+              <p>This is a header label</p>
+              <p>This is a header label</p>
+            </Stack>
+          }
+        >
+          This is a content
+        </AccordionSection>
+        <AccordionSection
+          id="0X1"
+          onExpand={action("onExpand")}
+          header={
+            <Stack justify="between">
+              <p>This is a header label</p>
+              <p>This is a header label</p>
+            </Stack>
+          }
+          footer={tmpFooter}
+        >
+          This is a content <br />
+          This is a content <br />
+          This is a content <br />
+          This is a content <br />
+          <br />
+          This is a content <br />
+          This is a content <br />
+          This is a content <br />
+          This is a content <br />
+          <br />
+          This is a content <br />
+          This is a content <br />
+          This is a content <br />
```

Comment:
Concerning stories, I recommend replacing placeholders with some real-world examples, then it's immediately clear what this component is good for. You can use the examples in your Figma design as inspiration.

Reply:
We'll solve this some other time.